src/styles/_variables.scss

```diff
@@ -0,0 +1,4 @@
+$colorBlack: #292929;
```

Comment:
The provided code seems to be following the rules mentioned in the task description. Great job! 🎉